spec/whois.md

```diff
@@ -0,0 +1,86 @@
+in learning more about the controller of the DID (the subject of the VCs or VPs).
+
+### `whois` Use Case
+
+The following is a use case for this capability. Consider an example of the
+`did:webs` controller being an educational institution that issues "degree"
+verifiable credentials to its graduate students. A graduate from the school
+submits as part of their job application to a company a [verifiable
+presentation] derived from the verifiable credential they received from the
+school. The company receiving the presentation can verify the cryptography of
+the presentation, but can they trust the school that issued the verifiable
+credential? If the school issued the verifiable credential using its `did:webs`
+DID, the company can resolve the DID (included in the verifiable presentation
+from the student), and verify the [[ref: KEL]]. Then it can look in the DID's
+`whois` folder where it might find a couple of useful VCs or VPs with the
+`did:webs` DID as the subject. For example:
+
+- A verifiable presentation derived from a credential issued by [GLEIF]
+  containing the [LEI] of the school.
+  - Since the company knows about GLEIF, they can automate the lookup to get
+    more information about the school -- where it is registered, etc.
+- A verifiable presentation derived from a credential issued by the "Association
+  of Colleges and Universities" for the jurisdiction of the school.
+  - Since they don't know about the Association for that jurisdiction, the company can
+    repeat the process for the issuer of _that_ credential. They might (for
+    example), learn that the Association has in its `did:webs` `whois` folder a
+    verifiable credential issued by the government of the jurisdiction, and
+    the company recognizes that government.
+- A verifiable presentation derived from a credential issued by US News about
+  the school's placement on the [US News Rankings of Colleges and Universities].
+
+Such checks can all be done with a handful of HTTPS requests and the processing
+of the verifiable presentations. The result is an efficient, verifiable
+credential-based, decentralized trust registries.
+
+[GLEIF]: https://gleif.org
+[https://www.usnews.com/best-colleges/rankings/national-universities]: https://www.usnews.com/education/best-global-universities
+
+### `whois` folder Conventions
+
+The `whois` folder, if it exists, MUST be directly below the web root folder of
+the DID. The [DID URL path] to the folder MUST be `<did:webs DID>/whois`.
+
+The HTTPS path to the `whois` folder MUST be `<HTTPS URL conversion of did:webs
+identifier>/whois`. The Web Server that serves the `did:webs` documents MUST
+respond to a request for the HTTPS path to the `whois` folder with a list of the
+files within the folder so that an entity looking for information about the
+controller of the `did:webs` DID can see what verifiable credentials or
+verifiable presentations are available for review.
```

Comment:
Presumably the main HTTP method we'd be using here is GET. I am slightly concerned about careless implementation of directory semantics. (OPTIONS might be another good verb to support, but if a server turns on PUT, DELETE, or POST, that could be a problem. WebDAV verbs would be a problem, too, I think.) Of course, that would be dumb on the webmaster's part... Do you think we need to add a cautionary note?

Reply:
From our meeting:
How files are published is outside the scope? We'll move this discussion to an issue https://github.com/dhh1128/did-method-webs/issues/20